.github/workflows/ccpp.yml

```diff
@@ -84,3 +84,45 @@ jobs:
             else \
               exit 0; \
             fi
+
+  build-and-check-older-gcc:
```

Comment:
I think this should be build-and-check-gcc4.8

Reply:
Fixed in [74078a6](https://github.com/Rust-GCC/gccrs/pull/1059/commits/74078a6242f73bb3c11d2985f71c30e5f2d1eb72)